### ISSUE TITLE
Small improvement: client_max_body_size >> 2MB

### ISSUE THREAD

**Issue by dereuromark** [alt3/cakebox#89]
the default client_max_body_size is way too low for any development.

It would be nice if we could ship with

    client_max_body_size 50M;

inside each generated sites-available conf file inside server{} block

**Comment by bravo-kernel** [comment]
I can easily agree to your proposed modifications. There are two ways to make (default) file changes:

1. using bash scripts from the `cakebox` project
2. using the `UpdateShell` from the `cakebox-console` project, https://github.com/alt3/cakebox-console/blob/dev/src/Shell/UpdateShell.php

I forgot about the second option to be honest but we added that to make PR'ing these sort of system tweaks easier, no bash skills needed. I think you will be able to propose a PR with your eyes closed. 

Running a `vagrant reload --provision` already runs the UpdateShell so it should just work.

**Comment by bravo-kernel** [comment]
IIRC the UpdateShell is launched here https://github.com/alt3/cakebox/blob/102b0de77e8b23797d38d66b980bdfbdd0e513a4/.cakebox/Vagrantfile.rb#L186

**Comment by bravo-kernel** [comment]
@dereuromark the Bake template for the generated vhosts is found here. Feel free to PR your update (and other tunes you might find useful).

https://github.com/alt3/cakebox-console/blob/dev/src/Template/Bake/vhost_nginx.ctp

**Comment by dereuromark** [comment]
Yeah https://github.com/alt3/cakebox-console/blob/dev/src/Template/Bake/vhost_nginx.ctp#L29 needs to be switched over to 7.2 once the upgrade is complete for example.

**Comment by bravo-kernel** [comment]
Not really, it is already handled during the upgrade (e.g. 5 ref replaced by either 7.1 or 7.2).

I was hinting at the 2MB request.

**Comment by dereuromark** [comment]
Gotcha

PS: Interesting, why it didnt fix it for me. But all right, I will retry.

**Comment by bravo-kernel** [comment]
Probably due to a f** up on my behalf during progress but it is 100% reproducable for me now, works every single time.

If you have other improvements besides the 2MB feel free to stick them in the template.

**Comment by dereuromark** [comment]
Well, the defaults for session and php post limit size in general are bad for developing :)
I have to adjust them every time too, but maybe out of scope.

```
sudo sed -i 's|upload_max_filesize = 2M|upload_max_filesize = 128M|g' /etc/php/7.2/fpm/php.ini
sudo sed -i 's|post_max_size = 8M|post_max_size = 128M|g' /etc/php/7.2/fpm/php.ini
sudo sed -i 's|session.gc_maxlifetime = 1440|session.gc_maxlifetime = 144000|g' /etc/php/7.2/fpm/php.ini
sudo sed -i 's|memory_limit = 128M|memory_limit = 256M|g' /etc/php/7.2/fpm/php.ini
```

**Comment by bravo-kernel** [comment]
I see, do these setting work on the (per) vhost level as well? If so it would be nice to add them all there. If people don't like them they can easily adjust per site.

**Comment by dereuromark** [comment]
I dont think so, those need to be in the main php.ini for both CLI and WEB to work.

**Comment by bravo-kernel** [comment]
OK, I will put this on the list for the next fix.
